concordia/settings_dev.py

```diff
@@ -21,10 +21,30 @@
 
 CONCORDIA = {"netloc": "http://0.0.0.0:8000"}
 
-IMPORTER = {"IMAGES_FOLDER": "/tmp/concordia_images/"}
-
 S3_BUCKET_NAME = "concordia-staticpages"
 
+DEFAULT_FILE_STORAGE = "storages.backends.s3boto3.S3Boto3Storage"
+AWS_STORAGE_BUCKET_NAME = S3_BUCKET_NAME
+AWS_DEFAULT_ACL = None  # Don't set an ACL on the files, inherit the bucket ACLs
+
+AWS_ACCESS_KEY_ID = os.getenv("AWS_ACCESS_KEY_ID")
+AWS_SECRET_ACCESS_KEY = os.getenv("AWS_SECRET_ACCESS_KEY")
+
+IMPORTER = {
+    # /concordia_images is a docker volume shared by importer and concordia
+    "IMAGES_FOLDER": "/concordia_images/",
+    "S3_BUCKET_NAME": S3_BUCKET_NAME,
+}
+
+MEDIA_URL = "https://%s.s3.amazonaws.com/" % S3_BUCKET_NAME
+
+AWS_S3 = {
```

Comment:
We should remove this - it's an artifact of the old code

Reply:
It looks like this is actually used in serializers.py

Reply:
Those are also subject to removal – we could correct the media URL handling there but I'd want to check since most of those serializers are no longer called by a view in use. It's fine to defer that to a separate cleanup ticket, too.